Brokerages/Zerodha/ZerodhaBrokerageFactory.cs

```diff
@@ -86,6 +86,7 @@ public override IBrokerage CreateBrokerage(Packets.LiveNodePacket job, IAlgorith
                );
             //Add the brokerage to the composer to ensure its accessible to the live data feed.
             Composer.Instance.AddPart<IDataQueueHandler>(brokerage);
+            Composer.Instance.AddPart<IDataQueueUniverseProvider>(brokerage);
```

Comment:
nit - I don't think we required `IDataQueueUniverseProvider` to be added to the composer

Reply:
right, should be reverted I think